busy_beaver/__init__.py

```diff
@@ -48,4 +48,6 @@
 
 logger.info("[BusyBeaver] Starting Server")
 from .app import create_app  # noqa
+
+app = create_app()
```

Comment:
If you take this out, tests will pass. Having this running and then calling tests enables the task queue to run in async mode versus sync mode for tests.

I use `gunicorn` to call `create_app()` directly in the `entrypoint.sh` script

Reply:
Updated!